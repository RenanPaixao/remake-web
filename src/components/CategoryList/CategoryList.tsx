import { Button, SimpleGrid } from '@chakra-ui/react'
import { FaBucket, FaComputer, FaSheetPlastic } from 'react-icons/fa6'
import { GiSteelClaws } from 'react-icons/gi'
import { RiCopperCoinLine } from 'react-icons/ri'
import { LiaDrumSteelpanSolid, LiaWineGlassSolid } from 'react-icons/lia'
import { FaBoxOpen, FaRegNewspaper } from 'react-icons/fa'
import { useNavigate } from 'react-router-dom'
import { useTranslation } from 'react-i18next'

export const CategoryList = () => {
  const { t } = useTranslation()

  const iconSize = '2rem'
  const categories = [
    {
      icon: <FaComputer size={iconSize}/>,
      title: t('home.electronics')
    },
    {
      icon: <FaSheetPlastic size={iconSize}/>,
      title: t('home.plastic')
    },
    {
      icon: <GiSteelClaws size={iconSize}/>,
      title: t('home.steel')
    },
    {
      icon: <RiCopperCoinLine size={iconSize}/>,
      title: t('home.copper')
    },
    {
      icon: <LiaDrumSteelpanSolid size={iconSize}/>,
      title: t('home.aluminum')
    },
    {
      icon: <FaBucket size={iconSize}/>,
      title: t('home.buckets')
    },
    {
      icon: <LiaWineGlassSolid size={iconSize}/>,
      title: t('home.glass')
    },
    {
      icon: <FaRegNewspaper size={iconSize}/>,
      title: t('home.paper')
    },
    {
      icon: <FaBoxOpen size={iconSize}/>,
      title: t('home.boxes')
    }
  ]

  const navigate = useNavigate()

  /**
   * Redirects the user to the collect-places page with the given category.
   * @param title
   */
  function goToCategory(title: string) {
    navigate(`/collect-places/?tags=${title}`)
  }

  return <SimpleGrid columns={[2, 2, 3]} spacing='2rem' >
    {
      categories.map(({ title, icon }) => (
        <Button
          key={title}
          leftIcon={icon} p='2rem'
          onClick={() => goToCategory(title)}
        >
          {title}
        </Button>
      ))
    }
  </SimpleGrid>
}
