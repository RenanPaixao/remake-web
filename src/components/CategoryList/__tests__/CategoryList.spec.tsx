import { expect } from 'vitest'
import { customRender, testingHistory } from '../../../../tests/test-utils.tsx'
import { CategoryList } from '../CategoryList.tsx'

const mockPushHistory = vi.fn()
testingHistory.push = mockPushHistory

describe('CategoryList', () => {
  it('should render', async () => {
    const { container } = customRender(<CategoryList/>)

    expect(container).toMatchSnapshot()
  })
})
