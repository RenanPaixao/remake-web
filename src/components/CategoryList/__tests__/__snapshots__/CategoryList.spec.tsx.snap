// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CategoryList > should render 1`] = `
<div>
  <div
    class="css-1aj87wr"
  >
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 640 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M384 96V320H64L64 96H384zM64 32C28.7 32 0 60.7 0 96V320c0 35.3 28.7 64 64 64H181.3l-10.7 32H96c-17.7 0-32 14.3-32 32s14.3 32 32 32H352c17.7 0 32-14.3 32-32s-14.3-32-32-32H277.3l-10.7-32H384c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64zm464 0c-26.5 0-48 21.5-48 48V432c0 26.5 21.5 48 48 48h64c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48H528zm16 64h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H544c-8.8 0-16-7.2-16-16s7.2-16 16-16zm-16 80c0-8.8 7.2-16 16-16h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H544c-8.8 0-16-7.2-16-16zm32 160a32 32 0 1 1 0 64 32 32 0 1 1 0-64z"
          />
        </svg>
      </span>
      Electronics
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 384 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 448c0 35.3 28.7 64 64 64H224V384c0-17.7 14.3-32 32-32H384V64c0-35.3-28.7-64-64-64H64C28.7 0 0 28.7 0 64V448zM171.3 75.3l-96 96c-6.2 6.2-16.4 6.2-22.6 0s-6.2-16.4 0-22.6l96-96c6.2-6.2 16.4-6.2 22.6 0s6.2 16.4 0 22.6zm96 32l-160 160c-6.2 6.2-16.4 6.2-22.6 0s-6.2-16.4 0-22.6l160-160c6.2-6.2 16.4-6.2 22.6 0s6.2 16.4 0 22.6zM384 384H256V512L384 384z"
          />
        </svg>
      </span>
      Plastic
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20.11 16.705h120.31l300.66 207.21 56.39 134-138.88-96-7.06-16.79zM309 423.295l-56.39-134-238.08-164.09v94.45zm-48.47-146.43l10.79 25.64 128.76 89-56.39-134-329.16-226.8v76.64z"
          />
        </svg>
      </span>
      Steel
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12.0049 22.0029C6.48204 22.0029 2.00488 17.5258 2.00488 12.0029C2.00488 6.48008 6.48204 2.00293 12.0049 2.00293C17.5277 2.00293 22.0049 6.48008 22.0049 12.0029C22.0049 17.5258 17.5277 22.0029 12.0049 22.0029ZM12.0049 20.0029C16.4232 20.0029 20.0049 16.4212 20.0049 12.0029C20.0049 7.58465 16.4232 4.00293 12.0049 4.00293C7.5866 4.00293 4.00488 7.58465 4.00488 12.0029C4.00488 16.4212 7.5866 20.0029 12.0049 20.0029ZM12.0049 7.05318L16.9546 12.0029L12.0049 16.9527L7.05514 12.0029L12.0049 7.05318ZM12.0049 9.88161L9.88356 12.0029L12.0049 14.1243L14.1262 12.0029L12.0049 9.88161Z"
          />
        </svg>
      </span>
      Copper
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 32 32"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M 16 6 C 8.82 6 3 8.462 3 11.5 L 3 20.5 C 3 23.538 8.82 26 16 26 C 23.18 26 29 23.538 29 20.5 L 29 11.5 C 29 8.462 23.18 6 16 6 z M 16 8 C 16.893125 8 17.734229 8.0394687 18.523438 8.109375 C 18.687739 8.1239284 18.84026 8.1449303 19 8.1621094 L 19 8.5996094 C 19 9.9196094 17.65 11 16 11 C 14.35 11 13 9.9196094 13 8.5996094 L 13 8.1621094 C 13.927038 8.0624121 14.92094 8 16 8 z M 12.896484 8.1738281 C 12.561764 8.2089911 12.240895 8.2528849 11.925781 8.3007812 C 12.062455 8.2799433 12.191484 8.2552358 12.332031 8.2363281 C 12.51362 8.2118992 12.70846 8.1948295 12.896484 8.1738281 z M 19.103516 8.1738281 C 19.29154 8.1948295 19.48638 8.2118992 19.667969 8.2363281 C 19.808516 8.2552358 19.937545 8.2799433 20.074219 8.3007812 C 19.759105 8.2528849 19.438236 8.2089911 19.103516 8.1738281 z M 11.523438 8.3632812 C 11.347689 8.3934993 11.181027 8.4293286 11.011719 8.4628906 C 11.102255 8.4452092 11.175073 8.4211401 11.267578 8.4042969 C 11.349705 8.3893433 11.439762 8.3775456 11.523438 8.3632812 z M 20.476562 8.3632812 C 20.560238 8.3775456 20.650295 8.3893433 20.732422 8.4042969 C 20.824927 8.4211401 20.897745 8.4452092 20.988281 8.4628906 C 20.818973 8.4293286 20.652311 8.3934993 20.476562 8.3632812 z M 10.519531 8.5761719 L 11.560547 10.380859 C 12.000547 11.120859 12.110625 11.990078 11.890625 12.830078 C 11.702439 13.49368 11.315254 14.067319 10.792969 14.486328 C 7.0213975 13.7066 5 12.230087 5 11.5 C 5 11.4365 5.0149426 11.366973 5.0449219 11.292969 C 5.3509502 10.537536 7.2506649 9.299294 10.519531 8.5761719 z M 21.480469 8.5761719 C 25.069561 9.3701328 27 10.788292 27 11.5 C 27 12.230212 24.977902 13.706709 21.205078 14.486328 C 20.683385 14.067598 20.297421 13.503084 20.109375 12.830078 C 19.889375 11.990078 19.999688 11.120859 20.429688 10.380859 L 21.480469 8.5761719 z M 16 13 C 17.31 13 18.399297 13.779375 18.779297 14.859375 C 19.157001 14.825038 19.516437 14.781496 19.869141 14.732422 C 18.710942 14.898479 17.424052 15 16 15 C 14.575948 15 13.289058 14.898479 12.130859 14.732422 C 12.483563 14.781496 12.842999 14.825038 13.220703 14.859375 C 13.600703 13.779375 14.69 13 16 13 z M 5 14.421875 C 7.3005188 15.96894 11.36057 17 16 17 C 20.63943 17 24.699481 15.96894 27 14.421875 L 27 20.5 C 27 21.516 23.145 24 16 24 C 8.855 24 5 21.516 5 20.5 L 5 14.421875 z M 11.113281 14.560547 C 11.278823 14.592606 11.447781 14.622055 11.619141 14.650391 C 11.446245 14.62166 11.279655 14.592163 11.113281 14.560547 z M 20.886719 14.560547 C 20.720345 14.592163 20.553755 14.62166 20.380859 14.650391 C 20.552219 14.622055 20.721177 14.592606 20.886719 14.560547 z"
          />
        </svg>
      </span>
      Aluminum
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 448 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M96 152v8H48v-8C48 68.1 116.1 0 200 0h48c83.9 0 152 68.1 152 152v8H352v-8c0-57.4-46.6-104-104-104H200C142.6 48 96 94.6 96 152zM0 224c0-17.7 14.3-32 32-32H416c17.7 0 32 14.3 32 32s-14.3 32-32 32h-5.1L388.5 469c-2.6 24.4-23.2 43-47.7 43H107.2c-24.6 0-45.2-18.5-47.7-43L37.1 256H32c-17.7 0-32-14.3-32-32z"
          />
        </svg>
      </span>
      Buckets
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 32 32"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M 8.3007812 5 L 8 5.5996094 C 7.4 6.8996094 7 8.7 7 11 C 7 15.6 10.5 19.400391 15 19.900391 L 15 26 L 10 26 L 10 28 L 22 28 L 22 26 L 17 26 L 17 19.900391 C 21.5 19.400391 25 15.6 25 11 C 25 8.7 24.6 6.8996094 24 5.5996094 L 23.800781 5 L 8.3007812 5 z M 9.5996094 7 L 22.400391 7 C 22.800391 8.2 23 9.6 23 11 C 23 13.1 22.099219 14.899219 20.699219 16.199219 L 20.599609 16.300781 C 20.499609 16.400781 20.400781 16.5 20.300781 16.5 C 20.200781 16.6 20.099609 16.599219 20.099609 16.699219 C 19.999609 16.799219 19.900781 16.800391 19.800781 16.900391 C 19.700781 17.000391 19.6 16.999609 19.5 17.099609 C 19.4 17.099609 19.300781 17.199219 19.300781 17.199219 C 19.200781 17.299219 19.100391 17.300391 18.900391 17.400391 C 18.800391 17.400391 18.699219 17.5 18.699219 17.5 C 18.599219 17.6 18.400781 17.599609 18.300781 17.599609 C 18.200781 17.599609 18.199609 17.699219 18.099609 17.699219 C 17.999609 17.699219 17.799219 17.800781 17.699219 17.800781 C 17.599219 17.800781 17.6 17.800391 17.5 17.900391 L 16.900391 17.900391 L 16.699219 17.900391 L 16 17.900391 C 12.1 17.900391 9 14.800391 9 10.900391 L 9 9.8007812 L 9 9.4003906 L 9.5996094 7 z"
          />
        </svg>
      </span>
      Glass
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 576 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M552 64H112c-20.858 0-38.643 13.377-45.248 32H24c-13.255 0-24 10.745-24 24v272c0 30.928 25.072 56 56 56h496c13.255 0 24-10.745 24-24V88c0-13.255-10.745-24-24-24zM48 392V144h16v248c0 4.411-3.589 8-8 8s-8-3.589-8-8zm480 8H111.422c.374-2.614.578-5.283.578-8V112h416v288zM172 280h136c6.627 0 12-5.373 12-12v-96c0-6.627-5.373-12-12-12H172c-6.627 0-12 5.373-12 12v96c0 6.627 5.373 12 12 12zm28-80h80v40h-80v-40zm-40 140v-24c0-6.627 5.373-12 12-12h136c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H172c-6.627 0-12-5.373-12-12zm192 0v-24c0-6.627 5.373-12 12-12h104c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H364c-6.627 0-12-5.373-12-12zm0-144v-24c0-6.627 5.373-12 12-12h104c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H364c-6.627 0-12-5.373-12-12zm0 72v-24c0-6.627 5.373-12 12-12h104c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H364c-6.627 0-12-5.373-12-12z"
          />
        </svg>
      </span>
      Paper
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 640 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M425.7 256c-16.9 0-32.8-9-41.4-23.4L320 126l-64.2 106.6c-8.7 14.5-24.6 23.5-41.5 23.5-4.5 0-9-.6-13.3-1.9L64 215v178c0 14.7 10 27.5 24.2 31l216.2 54.1c10.2 2.5 20.9 2.5 31 0L551.8 424c14.2-3.6 24.2-16.4 24.2-31V215l-137 39.1c-4.3 1.3-8.8 1.9-13.3 1.9zm212.6-112.2L586.8 41c-3.1-6.2-9.8-9.8-16.7-8.9L320 64l91.7 152.1c3.8 6.3 11.4 9.3 18.5 7.3l197.9-56.5c9.9-2.9 14.7-13.9 10.2-23.1zM53.2 41L1.7 143.8c-4.6 9.2.3 20.2 10.1 23l197.9 56.5c7.1 2 14.7-1 18.5-7.3L320 64 69.8 32.1c-6.9-.8-13.5 2.7-16.6 8.9z"
          />
        </svg>
      </span>
      Boxes
    </button>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;

exports[`Login > should render 1`] = `
<div>
  <div
    class="css-1aj87wr"
  >
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 640 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M384 96V320H64L64 96H384zM64 32C28.7 32 0 60.7 0 96V320c0 35.3 28.7 64 64 64H181.3l-10.7 32H96c-17.7 0-32 14.3-32 32s14.3 32 32 32H352c17.7 0 32-14.3 32-32s-14.3-32-32-32H277.3l-10.7-32H384c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64zm464 0c-26.5 0-48 21.5-48 48V432c0 26.5 21.5 48 48 48h64c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48H528zm16 64h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H544c-8.8 0-16-7.2-16-16s7.2-16 16-16zm-16 80c0-8.8 7.2-16 16-16h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H544c-8.8 0-16-7.2-16-16zm32 160a32 32 0 1 1 0 64 32 32 0 1 1 0-64z"
          />
        </svg>
      </span>
      Electronics
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 384 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 448c0 35.3 28.7 64 64 64H224V384c0-17.7 14.3-32 32-32H384V64c0-35.3-28.7-64-64-64H64C28.7 0 0 28.7 0 64V448zM171.3 75.3l-96 96c-6.2 6.2-16.4 6.2-22.6 0s-6.2-16.4 0-22.6l96-96c6.2-6.2 16.4-6.2 22.6 0s6.2 16.4 0 22.6zm96 32l-160 160c-6.2 6.2-16.4 6.2-22.6 0s-6.2-16.4 0-22.6l160-160c6.2-6.2 16.4-6.2 22.6 0s6.2 16.4 0 22.6zM384 384H256V512L384 384z"
          />
        </svg>
      </span>
      Plastic
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20.11 16.705h120.31l300.66 207.21 56.39 134-138.88-96-7.06-16.79zM309 423.295l-56.39-134-238.08-164.09v94.45zm-48.47-146.43l10.79 25.64 128.76 89-56.39-134-329.16-226.8v76.64z"
          />
        </svg>
      </span>
      Steel
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12.0049 22.0029C6.48204 22.0029 2.00488 17.5258 2.00488 12.0029C2.00488 6.48008 6.48204 2.00293 12.0049 2.00293C17.5277 2.00293 22.0049 6.48008 22.0049 12.0029C22.0049 17.5258 17.5277 22.0029 12.0049 22.0029ZM12.0049 20.0029C16.4232 20.0029 20.0049 16.4212 20.0049 12.0029C20.0049 7.58465 16.4232 4.00293 12.0049 4.00293C7.5866 4.00293 4.00488 7.58465 4.00488 12.0029C4.00488 16.4212 7.5866 20.0029 12.0049 20.0029ZM12.0049 7.05318L16.9546 12.0029L12.0049 16.9527L7.05514 12.0029L12.0049 7.05318ZM12.0049 9.88161L9.88356 12.0029L12.0049 14.1243L14.1262 12.0029L12.0049 9.88161Z"
          />
        </svg>
      </span>
      Copper
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 32 32"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M 16 6 C 8.82 6 3 8.462 3 11.5 L 3 20.5 C 3 23.538 8.82 26 16 26 C 23.18 26 29 23.538 29 20.5 L 29 11.5 C 29 8.462 23.18 6 16 6 z M 16 8 C 16.893125 8 17.734229 8.0394687 18.523438 8.109375 C 18.687739 8.1239284 18.84026 8.1449303 19 8.1621094 L 19 8.5996094 C 19 9.9196094 17.65 11 16 11 C 14.35 11 13 9.9196094 13 8.5996094 L 13 8.1621094 C 13.927038 8.0624121 14.92094 8 16 8 z M 12.896484 8.1738281 C 12.561764 8.2089911 12.240895 8.2528849 11.925781 8.3007812 C 12.062455 8.2799433 12.191484 8.2552358 12.332031 8.2363281 C 12.51362 8.2118992 12.70846 8.1948295 12.896484 8.1738281 z M 19.103516 8.1738281 C 19.29154 8.1948295 19.48638 8.2118992 19.667969 8.2363281 C 19.808516 8.2552358 19.937545 8.2799433 20.074219 8.3007812 C 19.759105 8.2528849 19.438236 8.2089911 19.103516 8.1738281 z M 11.523438 8.3632812 C 11.347689 8.3934993 11.181027 8.4293286 11.011719 8.4628906 C 11.102255 8.4452092 11.175073 8.4211401 11.267578 8.4042969 C 11.349705 8.3893433 11.439762 8.3775456 11.523438 8.3632812 z M 20.476562 8.3632812 C 20.560238 8.3775456 20.650295 8.3893433 20.732422 8.4042969 C 20.824927 8.4211401 20.897745 8.4452092 20.988281 8.4628906 C 20.818973 8.4293286 20.652311 8.3934993 20.476562 8.3632812 z M 10.519531 8.5761719 L 11.560547 10.380859 C 12.000547 11.120859 12.110625 11.990078 11.890625 12.830078 C 11.702439 13.49368 11.315254 14.067319 10.792969 14.486328 C 7.0213975 13.7066 5 12.230087 5 11.5 C 5 11.4365 5.0149426 11.366973 5.0449219 11.292969 C 5.3509502 10.537536 7.2506649 9.299294 10.519531 8.5761719 z M 21.480469 8.5761719 C 25.069561 9.3701328 27 10.788292 27 11.5 C 27 12.230212 24.977902 13.706709 21.205078 14.486328 C 20.683385 14.067598 20.297421 13.503084 20.109375 12.830078 C 19.889375 11.990078 19.999688 11.120859 20.429688 10.380859 L 21.480469 8.5761719 z M 16 13 C 17.31 13 18.399297 13.779375 18.779297 14.859375 C 19.157001 14.825038 19.516437 14.781496 19.869141 14.732422 C 18.710942 14.898479 17.424052 15 16 15 C 14.575948 15 13.289058 14.898479 12.130859 14.732422 C 12.483563 14.781496 12.842999 14.825038 13.220703 14.859375 C 13.600703 13.779375 14.69 13 16 13 z M 5 14.421875 C 7.3005188 15.96894 11.36057 17 16 17 C 20.63943 17 24.699481 15.96894 27 14.421875 L 27 20.5 C 27 21.516 23.145 24 16 24 C 8.855 24 5 21.516 5 20.5 L 5 14.421875 z M 11.113281 14.560547 C 11.278823 14.592606 11.447781 14.622055 11.619141 14.650391 C 11.446245 14.62166 11.279655 14.592163 11.113281 14.560547 z M 20.886719 14.560547 C 20.720345 14.592163 20.553755 14.62166 20.380859 14.650391 C 20.552219 14.622055 20.721177 14.592606 20.886719 14.560547 z"
          />
        </svg>
      </span>
      Aluminum
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 448 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M96 152v8H48v-8C48 68.1 116.1 0 200 0h48c83.9 0 152 68.1 152 152v8H352v-8c0-57.4-46.6-104-104-104H200C142.6 48 96 94.6 96 152zM0 224c0-17.7 14.3-32 32-32H416c17.7 0 32 14.3 32 32s-14.3 32-32 32h-5.1L388.5 469c-2.6 24.4-23.2 43-47.7 43H107.2c-24.6 0-45.2-18.5-47.7-43L37.1 256H32c-17.7 0-32-14.3-32-32z"
          />
        </svg>
      </span>
      Buckets
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 32 32"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M 8.3007812 5 L 8 5.5996094 C 7.4 6.8996094 7 8.7 7 11 C 7 15.6 10.5 19.400391 15 19.900391 L 15 26 L 10 26 L 10 28 L 22 28 L 22 26 L 17 26 L 17 19.900391 C 21.5 19.400391 25 15.6 25 11 C 25 8.7 24.6 6.8996094 24 5.5996094 L 23.800781 5 L 8.3007812 5 z M 9.5996094 7 L 22.400391 7 C 22.800391 8.2 23 9.6 23 11 C 23 13.1 22.099219 14.899219 20.699219 16.199219 L 20.599609 16.300781 C 20.499609 16.400781 20.400781 16.5 20.300781 16.5 C 20.200781 16.6 20.099609 16.599219 20.099609 16.699219 C 19.999609 16.799219 19.900781 16.800391 19.800781 16.900391 C 19.700781 17.000391 19.6 16.999609 19.5 17.099609 C 19.4 17.099609 19.300781 17.199219 19.300781 17.199219 C 19.200781 17.299219 19.100391 17.300391 18.900391 17.400391 C 18.800391 17.400391 18.699219 17.5 18.699219 17.5 C 18.599219 17.6 18.400781 17.599609 18.300781 17.599609 C 18.200781 17.599609 18.199609 17.699219 18.099609 17.699219 C 17.999609 17.699219 17.799219 17.800781 17.699219 17.800781 C 17.599219 17.800781 17.6 17.800391 17.5 17.900391 L 16.900391 17.900391 L 16.699219 17.900391 L 16 17.900391 C 12.1 17.900391 9 14.800391 9 10.900391 L 9 9.8007812 L 9 9.4003906 L 9.5996094 7 z"
          />
        </svg>
      </span>
      Glass
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 576 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M552 64H112c-20.858 0-38.643 13.377-45.248 32H24c-13.255 0-24 10.745-24 24v272c0 30.928 25.072 56 56 56h496c13.255 0 24-10.745 24-24V88c0-13.255-10.745-24-24-24zM48 392V144h16v248c0 4.411-3.589 8-8 8s-8-3.589-8-8zm480 8H111.422c.374-2.614.578-5.283.578-8V112h416v288zM172 280h136c6.627 0 12-5.373 12-12v-96c0-6.627-5.373-12-12-12H172c-6.627 0-12 5.373-12 12v96c0 6.627 5.373 12 12 12zm28-80h80v40h-80v-40zm-40 140v-24c0-6.627 5.373-12 12-12h136c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H172c-6.627 0-12-5.373-12-12zm192 0v-24c0-6.627 5.373-12 12-12h104c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H364c-6.627 0-12-5.373-12-12zm0-144v-24c0-6.627 5.373-12 12-12h104c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H364c-6.627 0-12-5.373-12-12zm0 72v-24c0-6.627 5.373-12 12-12h104c6.627 0 12 5.373 12 12v24c0 6.627-5.373 12-12 12H364c-6.627 0-12-5.373-12-12z"
          />
        </svg>
      </span>
      Paper
    </button>
    <button
      class="chakra-button css-16s31jg"
      type="button"
    >
      <span
        class="chakra-button__icon css-1wh2kri"
      >
        <svg
          aria-hidden="true"
          fill="currentColor"
          focusable="false"
          height="2rem"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 640 512"
          width="2rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M425.7 256c-16.9 0-32.8-9-41.4-23.4L320 126l-64.2 106.6c-8.7 14.5-24.6 23.5-41.5 23.5-4.5 0-9-.6-13.3-1.9L64 215v178c0 14.7 10 27.5 24.2 31l216.2 54.1c10.2 2.5 20.9 2.5 31 0L551.8 424c14.2-3.6 24.2-16.4 24.2-31V215l-137 39.1c-4.3 1.3-8.8 1.9-13.3 1.9zm212.6-112.2L586.8 41c-3.1-6.2-9.8-9.8-16.7-8.9L320 64l91.7 152.1c3.8 6.3 11.4 9.3 18.5 7.3l197.9-56.5c9.9-2.9 14.7-13.9 10.2-23.1zM53.2 41L1.7 143.8c-4.6 9.2.3 20.2 10.1 23l197.9 56.5c7.1 2 14.7-1 18.5-7.3L320 64 69.8 32.1c-6.9-.8-13.5 2.7-16.6 8.9z"
          />
        </svg>
      </span>
      Boxes
    </button>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
